src/Lykke.Service.BlockchainSettings.Console/BlockchainIntegrationSettings.json

```diff
@@ -4,51 +4,110 @@
       "Type": "LiteCoin",
       "ApiUrl": "http://litecoin-api.lykke-service.svc.cluster.local",
       "SignServiceUrl": "http://litecoin-sign.lykke-service.svc.cluster.local",
-      "SignFacadeUrl": "http://blockchain-sign-service-litecoin.lykke-service.svc.cluster.local",
-      "SignFacadeApiKey": "C7E5B840-9F40-4E5D-9CEC-2233048051CD",
-      "HotWalletAddress": "n4PZ2BYZMcQHjWjUnCQYqVfCTUBozVUcYj"
+      "HotWalletAddress": "mvqYvHSMbEEbQZ4GMnxfc7p5BurnJ95bWd",
```

Comment:
Why this file in the repo? Why settings read from the this file instead of setting service?